src/components/home/elements/TokenRecord.vue

```diff
@@ -49,6 +49,7 @@
             </div>
           </template>
           <b-dropdown-item @click="showModal">Get Token Info</b-dropdown-item>
+          <b-dropdown-item @click="verify">Verification</b-dropdown-item>
```

Comment:
The menu should check whether the token is already verified or not. If it is already verified, this item button should be hide.